docs/team/chao890.md

```diff
@@ -11,4 +11,59 @@ Given below are my contributions to the project.
 
 ## Contributions
 
-To be updated.
+### Summary of Contributions
+
+---
+
+#### Code Contributed
+
+#### Enhancements Implemented
+* **New Features**
+  * Implemented `add-leave` feature
+    * Created a `AddLeaveCommand` class to handle the logic (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/commands/AddLeaveCommand.java)) 
+    and `AddLeaveCommandParser` class to handle the parsing of the user input (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/parser/AddLeaveCommandParser.java)).
+    * Use Case: User can add leave periods for employees.
+    * Justification: With this command, users of Coydir will be able to keep track of employee leave periods and their availability at all times.
+    * Highlights: This command was slightly tricky to implement, as there were many cases to consider, such as overlapping leaves, or insufficient leaves. 
+  * Implemented `delete-leave` feature
+    * Created a `DeleteLeaveCommand` class to handle the logic (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/commands/DeleteLeaveCommand.java)) 
+    and `DeleteLeaveCommandParser` class to handle the parsing of the user input (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/parser/DeleteLeaveCommandParser.java)).
+    * Use Case: User can delete leave periods for employees.
+    * Justification: With this command, users of Coydir will be able to make changes easily when employees change their leave periods.
+    * Highlights: Smoother to implement, though just like `add-leave`, there were many dependencies and many files to be edited.
+
+
+* **Existing Features**
+  * Improved on the `Leave` class, by storing `Leave` objects in a priority queue, and adding methods to check for overlapping or equal leaves.
```

Comment:
Consider rephrasing: "Made improvements to `Leave` class by storing..." , "implemented methods to..."

also remove comma after class and queue.